.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,9 +30,15 @@ Feel free to delete this section.
 - [ ] Fill in the description [for developers](#for-developers) and [for customers](#for-customers) if relevant.
 - [ ] Ensure there are tests, or an explanation as to why they are not needed.
 - [ ] Ensure there is documentation, or an explanation as to why it is not needed.
-- [ ] Rebase on top of `main` branch before review and before merging.
+- [ ] Make the branch up to date with `main`.
 - [ ] Ensure that CI/CD passed.
 - [ ] Review the changes by yourself.
 - [ ] **Remove [this section](#checklist-before-review)**.
 
+## Checklist before merging
+
+- [ ] Make the branch up to date with `main`.
+- [ ] Rebase after merging #XX on the latest main: `git rebase --onto main "PREVIOUS_BRANCH_NAME"`
```

Comment:
`git rebase`? We are feeling brave… :grinning:

Reply:
It is in case of nested branching